test/show_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+
+class ShowTest < Minitest::Test
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+  end
+
+  def test_show_exists
+    assert_instance_of Show, @knight_rider
+  end
+
+  def test_attributes
+    assert_equal "Knight Rider", @knight_rider.name
+    assert_equal "Glen Larson", @knight_rider.creator
+  end
+
+  def test_show_characters
+    assert_equal [@michael_knight, @kitt], @knight_rider.characters
```

Comment:
this could get moved up into your attributes test since it's part of what got initialized